AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Storage/SQLite/StatementModelTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import SQLite
+import XCTest
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class StatementModelTests: XCTestCase {
+
+    func testDropLastPath_Root() throws {
```

Comment:
Can there be a testcase like "post." or empty string "" ?

Reply:
this is interesting to reason about, I'll add them for the reader to understand what is the outcome, but i don't think that input will happen